README.md

```diff
@@ -3,33 +3,70 @@
 for building and testing PHP extensions on Windows.
 
-## Example Usage
+## Inputs
+- `version`: the PHP version to build for
+  (`7.0`, `7.1`, `7.2`, `7.3`, `7.4`, `8.0` or `8.1`)
+- `arch`: the architecture to build for (`x64` or `x86`)
+- `ts`: thread-safety (`nts` or `ts`)
+- `deps`: (optional) dependency libraries to install; for now, only
+  [core](https://windows.php.net/downloads/php-sdk/deps/) and 
+  [PECL](https://windows.php.net/downloads/pecl/deps/) dependencies are available
+- `ignore_vs`: (optional) Ignore the VS version (`crt`) for given PECL dependencies
+- `devcmd`: (optional) set to anything other than `'true'` to disable the
+  [developer command propmt](https://github.com/ilammy/msvc-dev-cmd).
+- `sdkref`: (optional) The tag or SHA of the desired version of the 
+  [PHP-SDK binary tools](https://github.com/php/php-sdk-binary-tools);
+  e.g. `775cf0dbfafd8f563451f94d0d0a2a5d8a7ec623` or `php-sdk-2.2.0`
 
-````.yml
-- id: setup-php-sdk
-  uses: cmb69/setup-php-sdk@v0.5
+## Outputs
+- `toolset`: the required toolset version;
+  needs to be passed to the ilammy/msvc-dev-cmd action
+- `prefix`: the prefix of the PHP installation;
+  needs to be passed to configure
+- `vs`: the Visual Studio version (`crt`);
+  e.g. `vc15` or `vs16` 
+- `buildpath`: the build output path;
+  one of `x64\Release_TS`, `x64\Release`, `Release_TS` or `Release`
+- `file_tag`: a tag that consists of GitHub SHA/release tag, PHP version, ts, VS version and arch;
+  useful e.g. for naming files or archives in artifacts and release attachments: `f668a23-8.0-ts-vs16-x64` or `1.0.3-8.0-ts-vs16-x64`
+
+## Example Usage
+```yml
+- name: "Install PHP SDK"
+  id: sdk
+  uses: cmb69/setup-php-sdk@v0.7
   with:
     version: 8.0
     arch: x64
     ts: nts
-- uses: ilammy/msvc-dev-cmd@v1
-  with:
-    arch: x64
-    toolset: ${{steps.setup-php-sdk.outputs.toolset}}
+    
+# build    
 - run: phpize
-- run: configure --enable-dbase --with-prefix=${{steps.setup-php-sdk.outputs.prefix}}
+- run: configure --enable-dbase --with-prefix=${{steps.sdk.outputs.prefix}}
 - run: nmake
 - run: nmake test TESTS=tests
-````
+```
 
-## Inputs
+After the build you can use the provided output variables to package the files and attach them to a release or build artifacts:
+```yml
+- name: "Package"
+  run: |
+    md .artifact
+    copy ${{steps.sdk.outputs.buildpath}}\php_dbase.dll .artifact\php_dbase-${{steps.sdk.outputs.file_tag}}.dll
 
-- `version`: the PHP version to build for
-  (`7.0`, `7.1`, `7.2`, `7.3`, `7.4`, `8.0` or `8.1`)
-- `arch`: the architecture to build for (`x64` or `x86`)
-- `ts`: thread-safety (`nts` or `ts`)
-- `deps`: dependency libraries to install; for now, only
-  [core dependencies](https://windows.php.net/downloads/php-sdk/deps/) are available
+- name: "Upload artifacts"
+  uses: actions/upload-artifact@v3
+  if: contains(github.ref_type, 'branch')
+  with:
+    name: php_dbase-${{github.sha}}
```

Comment:
I'd worked on using setup-php-sdk recently: https://github.com/TysonAndre/var_representation/blob/main/.github/workflows/build_dlls.yml

I guess this part of the PR covers https://github.com/cmb69/setup-php-sdk/issues/18

1. Shouldn't this include the platform in the artifact name? e.g. `name: php_dbase-Windows-php-${{matrix.version}}-${{matrix.arch}}-${{matrix.ts}}-${{github.sha}}`
2. It might help users figure out which package to download if artifacts for Windows had Windows in the name.
3. In APCu, which I'd based my PR on, the LICENSE file was included with the artifact, which was a zip. And for the DLLs built by the Windows for PHP team, the README was also included in the zip, which also seems useful for an end user https://pecl.php.net/package/var_representation/0.1.1/windows 

    (The file names for the licenses and readmes will differ between projects, if they exist)

Reply:
> Shouldn't this include the platform in the artifact name?

It depends on how you prefer to name your artifacts. I personally prefer my `.dll`s to be versioned and package them [in a single artifact](https://github.com/codemasher/php-ext-xz/actions/runs/3254028257)

> It might help users figure out which package to download

Users should not dig in the build artifacts for downloads ever unless they want "nightly" or bleeding edge builds. In case you mean picking the right file from the releases - this is exactly what the example and the `file_tag` does. In case you want to to have the artifact named per version/arch/ts/vs, then move the `file_tag` there - black magick!

> the LICENSE file was included with the artifact

I trust whoever uses this example to be able to add a line `copy <dir>\LICENSE .artifact` (this seems to apply especially to those who chose the license that makes one reproduce a 47 paragraph disclaimer written in ALL CAPS...).
Same for the readme file, but i think a plain text readme would be better in that case.